lib/rubocop/cop/rails/reversible_migration.rb

```diff
@@ -246,6 +246,8 @@ def check_change_table_node(node, block)
         end
 
         def check_change_table_offense(receiver, node)
+          return unless node.send_type?
```

Comment:
Just for future reference, this could be fixed on L:241 by passing a type argument to `#each_child_node`. 🙂 

```ruby
block.each_child_node(:send) do |child_node|
```

Reply:
That make sense! Thank you for letting me know.